pages/Configuring/Dispatchers.md

```diff
@@ -85,6 +85,14 @@ layout pages (See the sidebar).
 
 {{< callout type=warning >}}
 
+[uwsm](../../Systemd-start) users should avoid using `exit` dispatcher, as exiting Hyprland this way causes various systemd-related problems after restarting the compositor, such xdg-desktop-portal-hyprland and other services do not restart. Use `loginctl terminate-user ""` instead.
+
+It's also strongly advised to replace the `exit` dispatcher with `loginctl terminate-user ""` inside `hyprland.conf` keybinds section.
```

Comment:
I still don't like this. Why not `loginctl terminate-user $(whoami)`?

Reply:
This is the command, listed on uwsm page and what worked for me.

Reply:
And, judging by @Vladimir-csp comment - `loginctl terminate-user $(whoami)` = `loginctl terminate-user ""` in this case.

Reply:
Yes, this clears my concern.